recipes/pyre/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyre" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: pyre
+    url: https://github.com/pyre/pyre/archive/v{{ version }}.tar.gz
+    sha256: d7bf203b4e1ae2c4f7989244306910babe809711181b20597e17226d874da4a0
+  - folder: config
+    url: https://github.com/aivazis/config/archive/3.5.0.tar.gz 
+    sha256: 00c5b2e75ce51261d78c7d0baea350c362e9b9cb2f57afd87e22163321229df4
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
+
+requirements:
+  build:
+      - {{ compiler('cxx') }}
+      - python
+  host:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyre
+    - journal
+
+about:
+  home: http://pyre.orthologue.com
+  license: BSD
```

Comment:
Which version of BSD? Also there should be a `license_family: BSD` entry

Reply:
Will update. Thanks